### PR TITLE
Fixed and added cp params

### PR DESCRIPTION
**Pull request by nisarg-soni**
- Added cp params for bandwidth rates, contract address, provider address, whitelisted addresses, blacklisted addresses
- fixed params for blacklist and whitelist

### CODE REVIEW

File: cmd/cp/cp.go [hunk 1]
```diff
@@ -83,7 +83,12 @@ func init() {
 	app.CreateCmd.ArgStore["rpc"]  = app.CreateCmd.Cmd.Flags().StringP("rpc", "u", "", "RPC url")
 	app.CreateCmd.ArgStore["regions"] = app.CreateCmd.Cmd.Flags().StringP("regions", "v", "ap-south-1", "Allowed AWS regions")
 	app.CreateCmd.ArgStore["rates"] = app.CreateCmd.Cmd.Flags().StringP("rates", "t", "", "Location of rates for instance types")
```
Comment (roshanr95) [inline cmd/cp/cp.go:85]: `instance-rates`

File: cmd/cp/cp.go [hunk 2]
```diff
@@ -83,7 +83,12 @@ func init() {
...
+	app.CreateCmd.ArgStore["bandwidth"] = app.CreateCmd.Cmd.Flags().StringP("bandwidth", "g", "", "Location of bandwidth rates file")
+	app.CreateCmd.ArgStore["provider"] = app.CreateCmd.Cmd.Flags().StringP("provider", "o", "", "Address of provider")
+	app.CreateCmd.ArgStore["contract"] = app.CreateCmd.Cmd.Flags().StringP("contract", "c", "", "Address of contract")
 	app.CreateCmd.ArgStore["black"] = app.CreateCmd.Cmd.Flags().StringP("black", "b", "", "Location of blacklist file")
 	app.CreateCmd.ArgStore["white"] = app.CreateCmd.Cmd.Flags().StringP("white", "w", "", "Location of whitelist file")
```
Comment (roshanr95) [inline cmd/cp/cp.go:90]: `image-whitelist`

**Review by roshanrags — CHANGES_REQUESTED** [review]
Name things properly everywhere.

**Comment by nisarg-soni** [comment]
> Name things properly everywhere.

All resolved

**Review by roshanrags — APPROVED** [review]

**Pull request merged**